evolution-chain/pallets/living-assets-evolution/src/lib.rs

```diff
@@ -227,10 +232,10 @@ impl<T: Config> Pallet<T> {
 		// we also use the last 12 bytes of the slot, since the first 4 bytes are always 0
 		bytes[..12].copy_from_slice(&slot_bytes[4..]);
 
-		let h160 = T::AccountIdToH160::convert(owner);
```

Comment:
`AccountIdToH160` should be removed from `Config` trait as well? since it's not used anymore, I think

Reply:
sgtm